.circleci/config.yml

```diff
@@ -526,7 +526,7 @@ jobs:
           artifact-type: "core"
       - publish-artifacts:
           artifact-type: "ui"
-#      - trigger-mobile-metrics
```

Comment:
Let's verify that this runs before enabling the production run

![Screen Shot 2021-05-20 at 8 23 47 AM](https://user-images.githubusercontent.com/3021882/119005944-c6411200-b944-11eb-8414-92fcdfeb95f3.png)